doc/index.rst

```diff
@@ -0,0 +1,48 @@
+.. py:currentmodule:: lsst.ts.weatherstation
+
+.. _lsst.ts.weatherstation:
+
+###########
+lsst.ts.eas
```

Comment:
eas -> weatherstation (and extend the ###s above and below to match)

Reply:
Thanks for catching that. Done.